5_databases/week1/homework

```diff
@@ -0,0 +1,55 @@
+-- Active: 1699216915341@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+ FROM task;
+
+
+
+
+SELECT COUNT(*) 
+FROM task 
+WHERE due_date <= CURDATE();
+
+
+
+SELECT *
+FROM task
+WHERE status_id= 3;
+
+
+SELECT *
+FROM task
+WHERE status_id!= 3;
+
+
+
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+
+SELECT MAX(created)
```

Comment:
This only returns the created field. Think more simple:

```
SELECT * 
FROM task 
ORDER BY created DESC LIMIT 1;
```

cool interpretation tho